packages/api/controllers/nomination.js

```diff
@@ -51,4 +51,25 @@ const createNomination = async (req, res) => {
   }
 };
 
-module.exports = { getNominationById, findAllNominataions, createNomination };
+const updateNomination = async (req, res) => {
+  const { id } = req.params;
+  try {
+    const nomination = await db.Nomination.update(
+      { status: req.body.status },
+      { where: { id } }
+    );
+    return res.status(204).json(nomination);
```

Comment:
This is conflicting, status 204 is a `No Content` but then you are returning actual json of the nomination. If you want to return the nomination, I would update to a 200 status. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200